src/core/bpf-util.c

```diff
@@ -13,17 +13,6 @@ bool cgroup_bpf_supported(void) {
         if (supported >= 0)
                 return supported;
 
-        r = cg_unified_controller(SYSTEMD_CGROUP_CONTROLLER);
```

Comment:
With this removed this is more like dlopen_bpf with cache than anything else. Maybe rename accordingly?

Reply:
I'd like to keep this as is at least in this PR.
I think we may need to update and move the check done in bpf_firewall_supported() here. Anyway, let's do that later.